doc/api/util.md

```diff
@@ -771,9 +771,13 @@ added: v8.0.0
 * {symbol} that can be used to declare custom promisified variants of functions,
 see [Custom promisified functions][].
 
-## Class: util.TextDecoder
+## Class: TextDecoder
 <!-- YAML
 added: v8.3.0
+changes:
+  - version: v11.0.0
```

Comment:
I'm not sure this should be specified here, I think we use a placeholder that gets filled in later?

Reply:
Yes, please use `REPLACEME` here.

Reply:
Same for the other v11.0.0 uses in this file and globals.md.